### PR TITLE
test: Update replacing suite

### PR DESCRIPTION
**Pull request by nalmabrcom**

### CODE REVIEW

File: tests/test_files/project/PAYLIB.CBL [hunk 1]
```diff
@@ -22,10 +22,10 @@
       * COPY REPL REPLACING ==TAG== BY ==ABC==
       * ==TAR== by ==XYZ==.
       * COPY REPL REPLACING ==TAG== BY DEF.
-       COPY PAYLIB REPLACING  A BY PAYROLL1
-                         B BY PAY-CODE
-                         C BY GROSS-PAY
-                         D BY HOURS.
+       COPY PAYLIB REPLACING LEADING  A BY PAYROLL1
```
Comment (temanbrcom) [inline tests/test_files/project/PAYLIB.CBL:25]: @ap891843 So, it is required to specify `LEADING` every time when the pattern is at the beginning of the line?

File: tests/cypress/integration/LSP/F97476.spec.js [hunk 2]
```diff
@@ -187,6 +191,12 @@ context('This is a F97476 spec', () => {
   describe('TC248135 REPLACING - check paragraph present', () => {
     beforeEach(() => {
       cy.updateConfigs('testing');
+      cy.updateConfigs('testing');
```
Comment (temanbrcom) [inline tests/cypress/integration/LSP/F97476.spec.js:194]: Is this line duplicated intentionally?

**Review by temanbrcom — COMMENTED** [review]
Why do you need an empty .txt file?

File: tests/test_files/project/PAYLIB.CBL (hunk 1 above)
Comment (ap891843) [reply tests/test_files/project/PAYLIB.CBL:25]: Yes. 
use leading clause to match starts with for a word.
use trailing to match ends with for a word.
or use delimited COBOL word, like :ABC: for exact match

**Comment by temanbrcom** [comment]
The first commit message doesn't match its content.

File: tests/cypress/integration/LSP/F97476.spec.js (hunk 2 above)
Comment (nalmabrcom) [reply tests/cypress/integration/LSP/F97476.spec.js]: Fixed. Duplicated accidentally.

**Review by temanbrcom — APPROVED** [review]
🐎

**Review by grianbrcom — APPROVED** [review]
:fox_face:

**Pull request merged**